src/app/docs/release-notes/[...slug]/page.jsx

```diff
@@ -30,25 +30,29 @@ export async function generateMetadata({ params }) {
   const { slug } = params;
 
   let label = '';
+  let releaseDate = '';
   let description = '';
   const currentSlug = slug.join('/');
   const isReleaseNotePage = RELEASE_NOTES_SLUG_REGEX.test(currentSlug);
```

Comment:
Let's not change anything related to release notes for the time being, since we will soon be working on a redesigned version for it. I think we will use a custom release notes og image instead of auto generated at the end.

Reply:
Got it, thanks for letting me know